coverage/artifacts/local_test.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package artifacts
+
+import (
+	"testing"
+
+	"k8s.io/test-infra/coverage/test"
+	"strings"
+)
+
+// generates coverage profile by running go test on target package
+func TestComposeCmdArgs(t *testing.T) {
+	profileDestinationPath := "./path/to/profile"
+	input := "."
+	expected := "test ./... -covermode=count -coverprofile " + profileDestinationPath
```

Comment:
Use a full string literal here -- don't re-use input variables. It should take two intentional actions to change this test -- edit the input, then the output.

Reply:
done